scheduler/src/cook/mesos/api.clj

```diff
@@ -240,6 +240,11 @@
    (s/optional-key :constraints) [Constraint]
    (s/optional-key :container) Container
    (s/optional-key :executor) (s/enum "cook" "mesos")
+   (s/optional-key :executor-log-level) (s/enum "DEBUG" "ERROR" "INFO" "WARNING")
+   (s/optional-key :executor-max-message-length) PosInt
```

Comment:
Do we really want to let users modify this?

Reply:
Please see https://github.com/twosigma/Cook/pull/468#discussion_r141377993